dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/AddDownloadableProductToCartTest.php

```diff
@@ -0,0 +1,337 @@
+<?php
```

Comment:
Tests should be in a separate namespace
See Configurable product for example

Reply:
**done**